source/usage-examples/command.txt

```diff
@@ -0,0 +1,48 @@
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+You can run all raw database operations using the
+``MongoDatabase.runCommand()`` method. Call the ``runCommand()`` method
+with your command object on an instance of a database for diagnostic and
+administrative tasks such as fetching server stats or initializing a
+replica set.
+
+.. note::
+    Use the :manual:`mongo shell </reference/mongo-shell>` for
+    administrative tasks instead of the Java Driver whenever possible.
+
+The ``runCommand()`` method accepts a :manual:`command
+<reference/command/>` in Bson format. By default, ``runCommand`` returns
+an object of type ``org.bson.Document`` containing the output of the
+database command.
+
+You can specify a :manual:`read preference <core/read-preference/>` or
```

Comment:
Same comment as above. Also, unless you think many of the commands they will run require read preference, let's omit it from the usage example. I think the current paragraph is also missing the ordering if they do not want a read preference but want a return type.

Reply:
"you can specify a read preference or a return type ... as an optional second parameter" covers both read preference and return type only, but I can trim this to just cover return type. Only issue I have there is that I hate including just *one* of the arguments arbitrarily, but I think you have a good point that read preference is not something that most users need to know about in this context.